storage/block/block.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/m3db/m3db/digest"
 	"github.com/m3db/m3db/ts"
 	xio "github.com/m3db/m3db/x/io"
+	m3dbTime "github.com/m3db/m3db/x/time"
```

Comment:
nit: could you change the casing here, `m3dbtime` instead of `m3dbTime`

Reply:
Oh yeah I forgot its always supposed to be lowercase